File: src/main/java/com/dh/ondot/schedule/application/command/QuickScheduleCommand.java
package com.dh.ondot.schedule.application.command;

import java.time.LocalDateTime;

public record QuickScheduleCommand(
        Long memberId,
        LocalDateTime appointmentAt,
        PlaceInfo departure,
        PlaceInfo arrival
) {
    public record PlaceInfo(String title, String roadAddress, Double longitude, Double latitude) {}
}

File: src/main/kotlin/com/dh/ondot/schedule/application/command/QuickScheduleCommand.kt
package com.dh.ondot.schedule.application.command

import java.time.LocalDateTime

data class QuickScheduleCommand(
    val memberId: Long,
    val appointmentAt: LocalDateTime,
    val departure: PlaceInfo,
    val arrival: PlaceInfo,
) {
    data class PlaceInfo(
        val title: String,
        val roadAddress: String,
        val longitude: Double,
        val latitude: Double,
    )
}
